core/src/test/java/com/gentics/mesh/core/schema/field/HtmlFieldMigrationTest.java

```diff
@@ -43,15 +40,6 @@ public void testRemove() throws Exception {
 		removeField(CREATEHTML, FILLTEXT, FETCH);
 	}
 
-	@Test
-	@Override
-	public void testRename() throws Exception {
```

Comment:
Check why this was removed.

Reply:
A custom script was used for renaming fields.